python/src/scipp/plot/slicer.py

```diff
@@ -295,20 +292,48 @@ def __init__(self,
                 dim]
             self.members["widgets"]["labels"][dim_str] = self.lab[dim]
 
-        if self.masks is not None:
+        # Add controls for masks
+        self.add_masks_controls()
+
+        return
+
+    def add_masks_controls(self):
+        # Add controls for masks
+        masks_found = False
+        for name, array in self.scipp_obj_dict.items():
+            self.masks[name] = {}
+            if len(array.masks) > 0:
+                masks_found = True
+                for key in array.masks:
+                    self.masks[name][key] = widgets.Checkbox(
+                        value=self.params["masks"][name]["show"],
+                        description="{}:{}".format(name, key),
```

Comment:
```suggestion
                        description=f"{name}:{key}",
```
would be more modern